PowerShellTools/DebugEngine/VSXHost.cs

```diff
@@ -25,6 +25,7 @@ namespace PowerShellTools.DebugEngine
     using PowerShellTools.Common.ServiceManagement.DebuggingContract;

```

Comment:
The was the original name, I guess it was named like this because:
1. partial class, so it may not able to be consistent with its class name as it was taken.
2. implemented the host related interfaces.